pkg/app/piped/platformprovider/ecs/client.go

```diff
@@ -277,29 +277,29 @@ func (c *client) ServiceExists(ctx context.Context, clusterName string, serviceN
 	return false, nil
 }
 
-func (c *client) GetListener(ctx context.Context, targetGroup types.LoadBalancer) (string, error) {
+func (c *client) GetListenersArn(ctx context.Context, targetGroup types.LoadBalancer) ([]string, error) {
```

Comment:
Would it be better naming?

```suggestion
func (c *client) GetListenerArns(ctx context.Context, targetGroup types.LoadBalancer) ([]string, error) {
```

Reply:
tbh, I like the current one but will change it as I accepted you point as well 😄